app/FreshRSS.php

```diff
@@ -90,7 +90,7 @@ public static function loadStylesAndScripts() {
 				}
 				$filetime = @filemtime(PUBLIC_PATH . '/themes/' . $theme_id . '/' . $filename);
 				$url = '/themes/' . $theme_id . '/' . $filename . '?' . $filetime;
-				header('Link: <' . Minz_Url::display($url, '', 'root') . '>;rel=preload', false);	//HTTP2
+				//header('Link: <' . Minz_Url::display($url, '', 'root') . '>;rel=preload', false);	//HTTP2
```

Comment:
If it's useless, it's better to remove it completely.

Reply:
Removed in https://github.com/FreshRSS/FreshRSS/pull/2040/commits/a666cd8307290995711c2f597d3e7354ac30e022
It might be useful associated with a smarter way of guessing whether the client needs it or not (e.g. first request)